packages/dom/src/serialize-cssom.js

```diff
@@ -104,9 +104,21 @@ export function serializeCSSOM(ctx) {
       /* istanbul ignore next: tested, but coverage is stripped */
       if (clone.constructor.name === 'HTMLDocument' || clone.constructor.name === 'DocumentFragment') {
         // handle document and iframe
-        clone.body.prepend(styleLink);
+        // We are checking if we have multiple stylesheets present for the same clone or clone.body then we add
+        // them in the same order in which we receive them.
+        const lastLink = [...clone.body.querySelectorAll('link[data-percy-adopted-stylesheets-serialized]')].pop();
+        if (lastLink) {
+          lastLink.after(styleLink);
+        } else {
+          clone.body.prepend(styleLink);
```

Comment:
why can we just use append directly? that should also fix this right?

Reply:
querySelector('link[data-percy-adopted-stylesheets-serialized]:last-of-type');
can you check if this works